app/code/core/Mage/Core/Controller/Front/Action.php

```diff
@@ -36,7 +36,7 @@ class Mage_Core_Controller_Front_Action extends Mage_Core_Controller_Varien_Acti
     /**
      * Session namespace to refer in other places
      */
-    const SESSION_NAMESPACE = 'frontend';
+    const SESSION_NAMESPACE = 'om_session';
```

Comment:
do we need strictly avoid `frontend` keyword?

If not, I would keep it as part of new cookie name to be easily recognizable, e.g. `om_frontend`